packages/story-editor/src/components/checklist/checklist.js

```diff
@@ -98,7 +98,6 @@ const ThroughputPopup = forwardRef(function ThroughputPopup(
             popupId={POPUP_ID}
           />
           <Tablist
-            id="pre-publish-checklist"
```

Comment:
Why, is this `Tablist` actually not unique? I noticed the change in the test selectors too.

Reply:
It _is_ unique, but in the e2e tests the checklist can be targeted in a better way that doesn't rely on the `id`. Hence I removed it.